src/Storefront/Resources/views/storefront/layout/cookie/cookie-configuration-child.html.twig

```diff
@@ -28,11 +28,11 @@
         <label
             for="cookie_{{ cookie }}"
             class="custom-control-label">
-            {{ snippetKeyName ? snippetKeyName|trans|sw_sanitize : cookie }}
+            {{ childName ? childName|trans|sw_sanitize : cookie }}
```

Comment:
should we already remove the `trans` to save some computing time? this should also be mentioned in the changelog